README.md

```diff
@@ -1,6 +1,8 @@
 <p align="center">
-  <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
-  <h3 align="center">Vercel</h3>
+  <a href="https://vercel.com">
+    <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
+    <h3 align="center">Vercel</h3>
```

Comment:
Does `color` work?

```suggestion
    <h3 align="center" style="color: black;">Vercel</h3>
```

Reply:
I have a feeling that arbitrary css doesn't work

Reply:
Ya, nvm. It's [not supported by GH](https://stackoverflow.com/q/11509830/376773). Also see: https://github.com/github/markup/issues/369